README.md

```diff
@@ -4,7 +4,7 @@ This extension allows you to pick a background image for the whole Visual Studi
 
 Available on [Visual Studio Gallery](https://marketplace.visualstudio.com/items?itemName=vs-publisher-593793.MoeIDE).
 
-Supported version: Visual Studio 2015, 2017 (or above if no breaking changes)
+Supported version: Visual Studio 2015, 2017, 2019 (or above if no breaking changes)
```

Comment:
Actually it should be "2015 and above", as the region written in manifest .

Reply:
Resolved this in 5e5bc882377a8a0b01b15d7dfa14a23dcb596fdb